app/modules.go

```diff
@@ -55,32 +56,38 @@ import (
 // ModuleEncodingRegisters keeps track of all the module methods needed to
 // register interfaces and specific type to encoding config
 var ModuleEncodingRegisters = []module.AppModuleBasic{
+	// cosmos-sdk std
 	auth.AppModuleBasic{},
-	genutil.AppModuleBasic{},
+	authzmodule.AppModuleBasic{},
 	bankModule{},
 	capability.AppModuleBasic{},
-	stakingModule{},
-	mintModule{},
+	circuitModule{},
+	consensus.AppModuleBasic{},
 	distribution.AppModuleBasic{},
+	evidence.AppModuleBasic{},
+	feegrantmodule.AppModuleBasic{},
+	genutil.AppModuleBasic{},
 	govModule{},
+	mintModule{},
```

Comment:
should this be moved down to be grouped with celestia modules? I think this is custom, correct?

Reply:
yeah technically

Reply:
yea the x/mint module is technically Celestia specific b/c it hard-codes the inflation schedule

Reply:
moved in https://github.com/celestiaorg/celestia-app/pull/4593/commits/53c09508d9cacfa6d55af9ef834aff58c1a2f054 and addressed linter complaints